packages/presentation/lib/src/pages/home_page/home.dart

```diff
@@ -37,7 +37,9 @@ class _HomeState extends BlocScreenState<Home, HomeBloc> {
         elevation: 0,
         actions: [
           IconButton(
-            onPressed: () {},
+            onPressed: () {
+              //todo implement
```

Comment:
why are you making this comment?)

Reply:
Yulia says that, that is a good practice, u will not forget to implement this in future